runner/MODULE.bazel

```diff
@@ -50,7 +50,7 @@ archive_override(
 # Go modules
 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
 go_deps.from_file(go_mod = "//:go.mod")
-use_repo(go_deps, "com_github_aspect_build_aspect_gazelle", "com_github_aspect_build_aspect_gazelle_language_js", "com_github_aspect_build_aspect_gazelle_language_orion", "com_github_bazelbuild_bazel_gazelle", "com_github_bazelbuild_buildtools", "com_github_engflow_gazelle_cc", "com_github_fatih_color", "com_github_go_git_go_git_v5", "com_github_pmezard_go_difflib", "in_gopkg_yaml_v3", "io_opentelemetry_go_otel", "io_opentelemetry_go_otel_trace", "org_golang_x_term")
```

Comment:
I am not understanding this issue atm and have worked around this manually by deleting these `use_repo` params, but invoking `bazel mod tidy` adds them back.

Issue: these `com_github_aspect_build_aspect_gazelle*` go modules are designed to be imported as bazel modules, not go modules. When imported as go modules their BUILD files are incorrect (I think because they are no longer loaded via bzlmod so the module scoping is incorrect?).